### PR TITLE
Docsp 8124

### PR DESCRIPTION
**Pull request by skerschb**

### CODE REVIEW

File: Dockerfile.xlarge [hunk 1]
```diff
@@ -33,10 +33,11 @@ WORKDIR /home/docsworker-xlarge
...
 RUN python3 -m pip uninstall -y snooty
 RUN python3 -m pip install --upgrade pip flit
-RUN git clone https://github.com/mongodb/snooty-parser.git snooty-parser && \
+RUN git clone https://github.com/mongodb/snooty-parser.git && \
 	cd snooty-parser && \
-	git fetch --all && \
-	git reset --hard origin/master && \
+	git fetch --tags && \
+	latestTag=$(git describe --tags `git rev-list --tags --max-count=1`) && \
+	git checkout $latestTag && \
```
Comment (i80and) [inline Dockerfile.xlarge]: In theory this should be safe, but it's generally a best practice to wrap shell variables in double-quotes; e.g. `git checkout "$latestTag"`

**Comment by madelinezec** [comment]
built and ran autobuilder, successfully completed job

File: Dockerfile.xlarge (hunk 1 above)
Comment (skerschb) [reply Dockerfile.xlarge]: changed, lemme know if ok.

**Comment by danielborowski** [comment]
lgtm

**Pull request merged**